### DISCUSSION TITLE
CCIP2 Resolver Full Specs

### DISCUSSION THREAD

**Discussion by sshmatrix** [namesys-eth/ccip2-eth-resources#22] [General]
This is a place for organising all details of the CCIP2 Resolver for the final docs. @0xc0de4c0ffee: the [older README version](https://github.com/namesys-eth/ccip2-eth-resources/blob/main/docs/README.md) is perhaps slightly outdated[?] Can you start adding bullet points in the comments here? I will expand them in detail in the main body down below using stuff from older README. You'll be able to outline the resolver better than me. I'll do the same for the client part. 

---

`<expand on bullets here>`

**Comment by sshmatrix** [comment]
`<add bullets here>`

**Comment by 0xc0de4c0ffee** [comment]
- default callback signed data format/prefix 
`<recordhash.selector, signer, signature, result>`
- Approved callback signed data format/prefix 
`<approved.selector, signer, signer_signature, owner_signature, result>`
